lib/node_modules/@stdlib/math/generics/random/sample/lib/validate.js

```diff
@@ -48,6 +50,12 @@ function validate( opts, options ) {
 			return new TypeError( 'invalid option. `probs` option must be an array of probabilities that sum to one. Option: `' + opts.probs + '`.' );
 		}
 	}
+	if ( options.hasOwnProperty( 'mutate' ) ) {
```

Comment:
Use the `@stdlib/utils/has-own-property` utility. Applies also to the other options.